microapps/ArWorldScaleApp/app/src/main/java/com/arcgismaps/toolkit/arworldscaleapp/screens/MainScreen.kt

```diff
@@ -96,7 +98,22 @@ fun MainScreen() {
     val graphicsOverlays = remember { listOf(GraphicsOverlay()) }
     val proxy = remember { WorldScaleSceneViewProxy() }
     var initializationStatus by rememberWorldScaleSceneViewStatus()
-    var trackingMode by remember { mutableStateOf<WorldScaleTrackingMode>(WorldScaleTrackingMode.Geospatial()) }
+    var trackingMode by rememberSaveable(
+        saver = Saver(
+            save = {
+                it.value.name
+            },
+            restore = {
+                val mode = when (it) {
+                    "Geospatial" -> WorldScaleTrackingMode.Geospatial()
+                    else -> WorldScaleTrackingMode.World()
+                }
+                mutableStateOf(mode)
+            }
+        )
+    ) {
+        mutableStateOf<WorldScaleTrackingMode>(WorldScaleTrackingMode.Geospatial())
```

Comment:
Should default to `World()`:
```suggestion
        mutableStateOf<WorldScaleTrackingMode>(WorldScaleTrackingMode.World())
```